landscape/how-to/install-landscape-server-on-microsoft-azure.md

```diff
@@ -0,0 +1,228 @@
+  "managedBy": null,
+  "name": "Landscape-rg",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+### Create public-ip address resource
+
+Create static IP address in the `Landscape-rg` resource group:
+```
+az network public-ip create \
+	--resource-group Landscape-rg \
+    --name LandscapePublicIP \
+    --location eastus \
+    --allocation-method Static
+```
+
+Output will be displayed in JSON format and will show static IP address (extract below):
+```
+"ipAddress": "34.139.255.120",
+"ipTags": [],
+"location": "eastus",
+"name": "LandscapePublicIP",
+"provisioningState": "Succeeded",
+"publicIPAddressVersion": "IPv4",
+"publicIPAllocationMethod": "Static",
+"resourceGroup": "Landscape-rg",
+```
+
+Copy the IP address and set it as the A record value for the domain or subdomain that will serve as the FQDN. You set the A record in your DNS service. You can also use Azure DNS zone to host your DNS domain and manage your DNS records.
+
+Verify the A record using `nslookup`. Replace `{landscape.domain.com}` with your FQDN:
+
+```
+nslookup {landscape.domain.com}
+```
+
+You’ll receive output similar to:
+```
+Server:		127.0.0.53
+Address:	127.0.0.53#53
+   
+Non-authoritative answer:
+Name:	landscape.domain.com
+Address: 34.139.255.120
+```
+
+If the address value in the `nslookup` output matches the value of the `LandscapePublicIP` static IP address, the LetsEncrypt SSL provisioning step defined in the cloud-init configuration automation template will succeed.
+
+
+## Deploy Landscape Server VM with cloud-init
+
+Before beginning the deployment process with cloud-init, you must choose which of the two cloud-init configuration automation templates you want to use. In the [Landscape Scripts](https://github.com/canonical/landscape-scripts) Github repository, there are two Landscape Quickstart cloud-init configuration templates: [`cloud-init-quickstart.yaml`](https://github.com/canonical/landscape-scripts/blob/main/provisioning/cloud-init-quickstart.yaml) and [`cloud-init-quickstart-fips.yaml`](https://github.com/canonical/landscape-scripts/blob/main/provisioning/cloud-init-quickstart-fips.yaml).
+
+The `cloud-init-quickstart.yaml` template is designed for anyone, and the `cloud-init-quickstart-fips.yaml` is designed for FIPS compliant deployments of Landscape Server. For more information, see [how to install FIPS hardened Landscape Server](https://ubuntu.com/landscape/docs/install-fips-hardened-landscape-server).
+
+Once you’ve chosen your configuration template, complete the following steps.
+
+1. Set the `IMAGE_FAMILY` environment variable based on the cloud-init configuration you chose.
+
+- If you’re using `cloud-init-quickstart.yaml`, run:
+```
+curl -s https://raw.githubusercontent.com/canonical/landscape-scripts/main/provisioning/cloud-init-quickstart.yaml -o cloud-init.yaml
+IMAGE_FAMILY=Canonical:0001-com-ubuntu-server-jammy:22_04-lts-gen2:latest
```

Comment:
I think this should point to Canonical:0001-com-ubuntu-pro-jammy:pro-22_04-lts-gen2:latest to be equivalent to the https://ubuntu.com/landscape/docs/install-on-google-cloud it is based on.
See https://documentation.ubuntu.com/azure/en/latest/azure-how-to/instances/find-ubuntu-images/

Reply:
Changed, see comment below